docs/sources/collect/choose-component.md

```diff
@@ -69,3 +69,12 @@ If your application isn't yet instrumented for tracing, use `beyla.ebpf` to gene
 ## Profiles
 
 Use `pyroscope.*` components to collect profiles.
+
+## Frontend telemetry
+
+Use `otelcol.receiver.faro` and `otelcol.exporter.faro` components to process telemetry data collected with the [Grafana Faro Web SDK][faro-sdk] before sending it to Grafana Cloud. This approach is particularly useful for [Frontend Observability][frontend-observability] users who want full control over their data processing pipeline, allowing you to: remove sensitive information, filter out specific signals, apply custom processing rules.
```

Comment:
```suggestion
Use `otelcol.receiver.faro` and `otelcol.exporter.faro` components to process telemetry data collected with the [Grafana Faro Web SDK][faro-sdk] before sending it to Grafana Cloud. 
This approach is particularly useful for [Frontend Observability][frontend-observability] users who want full control over their data processing pipeline.
This allows you to remove sensitive information, filter out specific signals, and apply custom processing rules.
```

Can you not do these things even with faro.receiver?

Reply:
Probably it is possible. For traces it is possible to use otlp processors. For logs we can use [loki.process](https://grafana.com/docs/alloy/latest/reference/components/loki/loki.process), [loki.secretfilter](https://grafana.com/docs/alloy/latest/reference/components/loki/loki.secretfilter) components. The experience is just different from otel